tests/Cart/AddToCart.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '../../src/fixtures/UsedFixtures';
+import { CartPage } from '../../src/pages/CartPage';
+import { MenuPage } from '../../src/pages/MenuPage';
+import { NavHeader } from '../../src/components/NavHeader';
+
+let navHeader: NavHeader;
+let menuPage: MenuPage;
+let cartPage: CartPage;
+
+test.describe('Test adding to cart via confirmation dialog', () => {
+  test.beforeEach(async({ page }) => {
+    navHeader = new NavHeader(page);
+    cartPage = new CartPage(page);
+    menuPage = new MenuPage(page);
+    await cartPage.open();
+  });
+
+  test('Adds coffee item to cart after clicking "Yes" in confirmation dialog', async ({page}) => {
+    let coffeeType = 'Espresso';
+    await navHeader.toMenu();
```

Comment:
I have no idea what method 'toMenu' does. The method should have a **verb** describing the action performed.

Reply:
okok, renamed 'toMenu -> goToMenu', 'toCart -> goToCart', 'toGitHub -> goToGitHub'